changelog.md

```diff
@@ -1,6 +1,14 @@
 # nimbusserver-ansible
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+# [2019.04] - 2019-04-09
```

Comment:
I'm looking at making the naming convention Year+R(number). This way it doesn't matter when we release it,
I'm going to confirm this with Keith.